### PR TITLE
Update install_locally to match the directory created in prepare_build_environment

### PR DESCRIPTION
**Pull request by bblacher**
The directory mentioned in install_locally doesn't match the one mentioned in prepare_build_environment.

This mr fixes this.

### CODE REVIEW

File: getting_started/setup_linux/install_locally.rst [hunk 1]
```diff
@@ -16,7 +16,7 @@ Make sure you have `set up your development environment <prepare_build_environme
 Build and install the kernel
 ****************************
 
-1. Change directory to ``~/sof/linux`` that you created on the setup page.
+1. Change directory to ``~/work/sof/linux`` that you created on the setup page.
```
Comment (plbossart) [inline getting_started/setup_linux/install_locally.rst:19]: @bblacher do you mind changing the other occurrences of the same issue? see the 'remove the kernel' section

Comment (bblacher) [reply getting_started/setup_linux/install_locally.rst:19]: Seems like I missed those. Fixed now.

**Review by plbossart — APPROVED** [review]
Thanks @bblacher

**Review by deb-intel — APPROVED** [review]

**Comment by lgirdwood** [comment]
@bblacher looks like 1 warning thta is blocking CI.
```
waiting for workers...
looking for now-outdated files... none found
pickling environment... done
checking consistency... done
preparing documents... done
writing output... [ 20%] algos/eq/equalizers_tuning .. architectures/firmware/sof-xtos/mem-mgmt
writing output... [ 40%] architectures/firmware/sof-xtos/overview .. developer_guides/add_new_arch
writing output... [ 60%] developer_guides/debugability/coredump-reader/index .. developer_guides/virtualization/virtualization
writing output... [ 80%] developer_guides/xtrun/index .. platforms/intel-legacy/index
writing output... [100%] platforms/intel-legacy/merrifield/index .. tsc/representatives

waiting for workers...
WARNING: dot code 'blockdiag edf_scheduling {\n\n\tnode_width = 250;\n\tnode_height = 120;\n\tdefault_fontsize = 16;\n\n\tComp_1 -> Comp_2\n\tcomment_1 -> Comp_2 [style=dashed]\n\tComp_2 -> Comp_3\n\tcomment_2 -> Comp_3 [style=dashed]\n\tComp_3 -> Comp_4\n\tcomment_3 -> Comp_4 [style=dashed]\n\tComp_4 -> sink\n\tcomment_4 -> sink [style=dashed]\n\n\tComp_1 [label="DP component 1\\n\n\t\t*processing period\\n\n\t\t*compute requirement"]\n\tComp_2 [label="DP component 2\\n\n\t\t*processing period\\n\n\t\t*compute requirement"]\n\tComp_3 [label="DP component 3\\n\n\t\t*processing period\\n\n\t\t*compute requirement"]\n\tComp_4 [label="DP component 4\\n\n\t\t*processing period\\n\n\t\t*compute requirement"]\n\n\tsink [label="real time sink", shape=endpoint, fontsize = 16]\n\n\tcomment_1 [label="DP1 to deliver data let\\n\n\t\tDP2 meet its objective"]\n\tcomment_2 [label="DP2 to deliver data let\\n\n\t\tDP3 meet its objective"]\n\tcomment_3 [label="DP3 to deliver data let\\n\n\t\tDP4 meet its objective"]\n\tcomment_4 [label="DP4 to deliver data\\n\n\t\tto real time-sink"]\n}\n': 'ImageDraw' object has no attribute 'textsize'
generating indices... genindex done
writing additional pages... search done
```

**Review by lgirdwood — APPROVED** [review]

**Comment by marc-hb** [comment]
> @bblacher looks like 1 warning that is showing in CI but its unrealted to this PR.

Unrelated indeed. It's triggered by brand new pillow version 10.0.0. Probably some subtle incompatibility. Fix submitted below, please review
- #472
- #473

Note this affects the sof daily builds too

https://github.com/thesofproject/sof/actions/workflows/daily-tests.yml

Screenshot 2023-07-10 at 17 01 56

**Pull request merged**